config/routes.rb

```diff
@@ -11,4 +11,7 @@
 	  root :to => 'users#dashboard', as: :authenticated
 	end
 	root :to => 'pages#landing'
+
+  get '/comments/:client_id', to: 'comments#client_comments'
```

Comment:
small thing: I think we should name this route: '/clients/:client_id/comments'

[http://www.restapitutorial.com/lessons/restfulresourcenaming.html](http://www.restapitutorial.com/lessons/restfulresourcenaming.html) for reference